flipperdevices/flipperzero-firmware
# [FL-1221] Hardware info CLI command

# What's new

- Add hw_info CLI command, printing hw version, flipper name and production date

# Verification 

- Build for f5 and f6 targets
- Verify hw_info cli command

# Checklist (do not modify)
